server/base/src/main/java/org/apache/accumulo/server/fs/VolumeManagerImpl.java

```diff
@@ -261,6 +263,12 @@ public FileSystem getFileSystemByPath(Path path) {
     }
   }
 
+  @Override
+  public RemoteIterator<LocatedFileStatus> listStatus(final Path path, final boolean recursive)
+      throws IOException {
+    return getFileSystemByPath(path).listFiles(path, recursive);
+  }
```

Comment:
The method should probably be called `listFiles` to match the underlying `FileSystem` API.

Reply:
Fixed in ed29ed2